### DISCUSSION TITLE
Security Onion installation stops on 64%

### DISCUSSION THREAD

**Discussion by tomaszniezbedny** [Security-Onion-Solutions/securityonion#5989] [Unsupported Versions]
Hi.

We have SO in standalone installation. /nsm directory was mapped to RAID0 array of two disks. We purchased additional storage to create RAID5. Then decided to reinstall SO but at the moment installation stops on 64%, below logs from sosetup.log:

so-aptcacherng
 so-nginx
 so-idstools
+so-elasticsearch
[INFO    ] Completed state [/opt/so/conf/so-status/so-status.conf] at time 22:20:33.131118 (duration_in_ms=10.376)
[INFO    ] Running state [/opt/so/conf/elasticsearch/so-elasticsearch-pipelines dcb04] at time 22:20:33.132203
[INFO    ] Executing state cmd.run for [/opt/so/conf/elasticsearch/so-elasticsearch-pipelines dcb04]
[INFO    ] Executing command '/opt/so/conf/elasticsearch/so-elasticsearch-pipelines' in directory '/root'
[INFO    ] {'pid': 28871, 'retcode': 0, 'stdout': "Waiting for ElasticSearch....................................................................................................................................................................................................................................................\nConnection attempt timed out.  Unable to connect to ElasticSearch.  \nPlease try: \n  -checking log(s) in /var/log/elasticsearch/\n  -running 'sudo docker ps' \n  -running 'sudo so-elastic-restart'\n\nLoading pipelines...\nbeats.common\n\ncommon\n\ncommon.nids\n\ndns.tld\n\nfilterlog\n\nhttp.status\n\nimport.wel\n\nlogscan.alert\n\nosquery.live_query\n\nosquery.normalize\n\nosquery.query_result\n\nossec\n\nstrelka.file\n\nsuricata.alert\n\nsuricata.common\n\nsuricata.dhcp\n\nsuricata.dnp3\n\nsuricata.dns\n\nsuricata.fileinfo\n\nsuricata.flow\n\nsuricata.ftp\n\nsuricata.ftp_data\n\nsuricata.http\n\nsuricata.ikev2\n\nsuricata.krb5\n\nsuricata.nfs\n\nsuricata.rdp\n\nsuricata.sip\n\nsuricata.smb\n\nsuricata.smtp\n\nsuricata.snmp\n\nsuricata.ssh\n\nsuricata.tftp\n\nsuricata.tls\n\nsyslog\n\nsysmon\n\nwin.eventlogs\n\nzeek.common\n\nzeek.common_ssl\n\nzeek.conn\n\nzeek.dce_rpc\n\nzeek.dhcp\n\nzeek.dnp3\n\nzeek.dns\n\nzeek.dpd\n\nzeek.files\n\nzeek.ftp\n\nzeek.http\n\nzeek.intel\n\nzeek.irc\n\nzeek.kerberos\n\nzeek.modbus\n\nzeek.mysql\n\nzeek.notice\n\nzeek.ntlm\n\nzeek.pe\n\nzeek.radius\n\nzeek.rdp\n\nzeek.rfb\n\nzeek.signatures\n\nzeek.sip\n\nzeek.smb_files\n\nzeek.smb_mapping\n\nzeek.smtp\n\nzeek.snmp\n\nzeek.socks\n\nzeek.software\n\nzeek.ssh\n\nzeek.ssl\n\nzeek.syslog\n\nzeek.tunnel\n\nzeek.tunnels\n\nzeek.weird\n\nzeek.x509", 'stderr': ''}
[INFO    ] Completed state [/opt/so/conf/elasticsearch/so-elasticsearch-pipelines dcb04] at time 22:24:39.650761 (duration_in_ms=246518.557)
[INFO    ] Running state [/usr/sbin/so-elasticsearch-templates-load] at time 22:24:39.651159
[INFO    ] Executing state cmd.run for [/usr/sbin/so-elasticsearch-templates-load]
[INFO    ] Executing command '/usr/sbin/so-elasticsearch-templates-load' in directory '/opt/so'
[INFO    ] {'pid': 31141, 'retcode': 0, 'stdout': "Waiting for ElasticSearch....................................................................................................................................................................................................................................................\nConnection attempt timed out.  Unable to connect to ElasticSearch.  \nPlease try: \n  -checking log(s) in /var/log/elasticsearch/\n  -running 'sudo docker ps' \n  -running 'sudo so-elastic-restart'\n\nLoading templates...\nso-beats\n\nso-common\n\nso-firewall\n\nso-flow\n\nso-ids\n\nso-import\n\nso-osquery\n\nso-ossec\n\nso-strelka\n\nso-syslog\n\nso-zeek", 'stderr': ''}
[INFO    ] Completed state [/usr/sbin/so-elasticsearch-templates-load] at time 07:35:09.185307 (duration_in_ms=33029534.146)
[INFO    ] Running state [/usr/sbin/so-elasticsearch-roles-load] at time 07:35:09.186057
[INFO    ] Executing state cmd.run for [/usr/sbin/so-elasticsearch-roles-load]
[INFO    ] Executing command '/usr/sbin/so-elasticsearch-roles-load' in directory '/opt/so'

I tried to execute so-elastic-restart, but did not help. Any ideas suggestions?

**Comment by dougburks** [comment]
After adding the RAID5 storage, did you perform a fresh OS installation?
